state-chain/custom-rpc/src/lib.rs

```diff
@@ -45,6 +46,38 @@ use std::{
 	sync::Arc,
 };
 
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct ScheduledSwap {
+	pub swap_id: SwapId,
+	pub base_asset: Asset,
+	pub quote_asset: Asset,
+	pub side: Side,
+	pub amount: NumberOrHex,
+	#[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Is this not the default behaviour? (skipping of None values)

Reply:
This is what the output looks like if I don't skip explicitly:
```
{"source_asset":null,"source_amount":null, ...}]
```
I imagine the product can still consume this, but last I heard we were going to skip the fields altogether.

Reply:
Yes, I think skipping is preferred, I'm just surprised we have to explicitly annotate this.